packages/react-core/src/components/Select/Select.tsx

```diff
@@ -164,8 +166,76 @@ const SelectBase: React.FunctionComponent<SelectProps & OUIAProps> = ({
   );
 };
 
-export const Select = React.forwardRef((props: SelectProps, ref: React.Ref<any>) => (
-  <SelectBase innerRef={ref} {...props} />
-));
+SelectBase.displayName = 'SelectBase';
+
+/**
+ * Extends SelectBaseProps
```

Comment:
Should expand on this a bit to explain when you'd use the SelectBaseProps over the simplified set. Or include the blurb somewhere else in the docs if we want to expand it to cover when to use select versus menu.